spec/sections/standard.adoc

```diff
@@ -463,7 +472,7 @@ The following table defines the `ouputs.files` object members.
 |`name`
 |Required
 2+|MUST be a string of only alphabetic, dash or underscore characters (defined by the regex† `^[a-zA-Z_-]+$`)
-indicating the key the processing system will place the file name captured for downstream processing.
+indicating the key the executor† will place the file name captured for downstream processing.
```

Comment:
Self-reviewing:  This sentence sounds terrible. Suggest something better.

Reply:
I would suggest a similar thing as above where you just say the name is used to refer to the output. I don't think it's necessary to talk about downstream processing.